src/GovUk.Education.ExploreEducationStatistics.Admin/Controllers/Api/Security/PermissionsController.cs

```diff
@@ -25,28 +25,38 @@ public class PermissionsController : ControllerBase
         private readonly IReleaseFileService _releaseFileService;
         private readonly IUserService _userService;
         private readonly IPreReleaseService _preReleaseService;
+        private readonly IUserPublicationRoleRepository _publicationRoleRepository;
+        private readonly IUserReleaseRoleRepository _releaseRoleRepository;
 
-        public PermissionsController(IPersistenceHelper<ContentDbContext> persistenceHelper,
+        public PermissionsController(
+            IPersistenceHelper<ContentDbContext> persistenceHelper,
             IReleaseFileService releaseFileService,
             IUserService userService,
-            IPreReleaseService preReleaseService)
+            IPreReleaseService preReleaseService, 
+            IUserPublicationRoleRepository publicationRoleRepository, 
+            IUserReleaseRoleRepository releaseRoleRepository)
         {
             _persistenceHelper = persistenceHelper;
             _releaseFileService = releaseFileService;
             _userService = userService;
             _preReleaseService = preReleaseService;
+            _publicationRoleRepository = publicationRoleRepository;
+            _releaseRoleRepository = releaseRoleRepository;
         }
 
         [HttpGet("permissions/access")]
         public async Task<ActionResult<GlobalPermissionsViewModel>> GetGlobalPermissions()
         {
+            var isBauUser = await _userService.CheckIsBauUser().IsRight();
+            
             return new GlobalPermissionsViewModel(
                 CanAccessSystem: await _userService.CheckCanAccessSystem().IsRight(),
                 CanAccessAnalystPages: await _userService.CheckCanAccessAnalystPages().IsRight(),
                 CanAccessAllImports: await _userService.CheckCanViewAllImports().IsRight(),
                 CanAccessPrereleasePages: await _userService.CheckCanAccessPrereleasePages().IsRight(),
                 CanManageAllTaxonomy: await _userService.CheckCanManageAllTaxonomy().IsRight(),
-                IsBauUser: await _userService.CheckIsBauUser().IsRight());
+                IsBauUser: isBauUser,
+                IsApprover: !isBauUser && (await IsReleaseApprover() || await IsPublicationApprover()));
```

Comment:
I've checked with Cam that he doesn't expect BAU users to see the tab ta.  I have included a comment here to explain why.  I think with this comment in place, I probably don't need to add equivalent comments to ReleaseService and MethodologyService, unless you feel strongly otherwise?

Reply:
I think it would be worth just adding a one liner - NOTE: BAU deliberately not returned all ready for approval or something.

My thinking is that someone could spot/forget that BAU is not handled in the future, then lose a fair chunk of time before they work out its deliberate.

Reply:
Yup, a comment is already added thanks!